qa/tasks/cephfs/mount.py

```diff
@@ -702,17 +702,10 @@ def run_python(self, pyscript, py_version='python3', sudo=False):
 
     def run_shell(self, args, timeout=300, **kwargs):
         kwargs.pop('omit_sudo', False)
-        sudo = kwargs.pop('sudo', False)
         cwd = kwargs.pop('cwd', self.mountpoint)
         stdout = kwargs.pop('stdout', StringIO())
         stderr = kwargs.pop('stderr', StringIO())
 
-        if sudo:
```

Comment:
A better way is to check `omit_sudo` here? Then you could avoid updating run_shell() with `sudo` and then `omit_sudo` they are basically the same thing.